app/src/conferences/templates/conferences/index.html

```diff
@@ -20,6 +20,10 @@ <h1 class="header center grey-text text-lighten-4 main_title">
               <h3 class="header center red-text text-lighten-2 main_title ">
                 {% trans "Registration is over" %}
               </h3>
+              {% elif zosia.registration_suspended %}
+              <h3 class="header center red-text text-lighten-2 main_title">
+                {% trans "Registration is suspended"%}
```

Comment:
I'd add more info, like: `"Registration is suspended, please wait for further information"`.